app-info-package-classic/package.json

```diff
@@ -5,8 +5,7 @@
   "description": "App Info package (classic)",
   "react-native": "src",
   "source": "src",
-  "main": "src",
-  "module": "src",
+  "exports": "./src/index",
```

Comment:
This is just a proposition because I am not entirely sure if it's better. I changed it, because I found out [here](https://esbuild.github.io/api/#main-fields-for-package-authors) that the `main`, `module` and `browser` package json fields are the old way of doing it. I have also read that the module field is not officially supported by the Node.js.

Reply:
Unfortunately `exports` field is not yet officially supported by Metro bundler:

- [Experimental support](https://facebook.github.io/metro/docs/package-exports/)
- [RFC for `exports` support](https://github.com/react-native-community/discussions-and-proposals/blob/main/proposals/0534-metro-package-exports-support.md)